unit2-java/lesson06.rst

```diff
@@ -4,14 +4,18 @@ Unit 2 (Java) - Lesson 6 Learning Objectives
 Goals
 -----
 
+- Understand the role of inheritance in enable effective code resuse.
+- Effectively use Java's inheritance tools to build clean, reusable code.
+
 Objectives
 ----------
 
 - Use the ``extends`` keyword to allow one class to inherit from another
-- Explain which fields and methods are inherited by a class extension, based on Java’s rules and taking into account access modifiers
+- Explain which fields and methods are inherited by a class extension, based on Java's rules and taking into account access modifiers
 - Call a parent class constructor from a child class constructor
 - Override methods when extending a class
-- Describe the role of the ``Object`` class in each class’ hierarchy
+- Describe the role of the ``Object`` class in each class' hierarchy
 - Describe and use abstract classes and abstract methods
 - Explain how variable and parameter declaration relates to inheritance (i.e. how inheritance enables polymorphic behavior)
 - Use basic class diagrams to visualize and design class relationships
+- Write unit tests to test Java code that uses inheritance
```

Comment:
"...tests to test..." kinda bothers me.  How about something like, "Write unit tests to verify Java code..." or, "Write tests for Java code..."